src/sources/audio_base_latency.ts

```diff
@@ -1,4 +1,12 @@
+import { isAndroid, isChromium } from '../utils/browser'
+
 export default function getAudioContextBaseLatency(): number | undefined {
+  // The signal emits warning in console, so is is enabled only on Android where it's less visible
+  const isAllowedPlatform = isAndroid() && isChromium()
+  if (!isAllowedPlatform) {
+    return undefined
+  }
```

Comment:
Let's distinguish missing API from disallowed browser. Like we do in the audio signal.